documentation/operators/src/nodes/network-requester-setup.md

```diff
@@ -151,102 +163,11 @@ Now that we have initialized our network-requester, we can start it with the fol
-
-### Upgrading to >= v1.1.10 from <v1.1.9
-
-In the previous version of the network-requester, users were required to run a nym-client along side it to function. As of `v1.1.10`, the network-requester now has a nym client embedded into the binary, so it can run standalone.
-
-If you are running an existing network requester registered with nym-connect, upgrading requires you move your old keys over to the new network requester configuration. We suggest following these instructions carefully to ensure a smooth transition.
-
-Initiate the new network requester:
-
-```
-nym-network-requester init --id mynetworkrequester
-```
-
-Copy the old keys from your client to the network-requester configuration that was created above:
-
-```
-cp -vr ~/.nym/clients/myoldclient/data/* ~/.nym/service-providers/network-requester/mynetworkrequester/data
-```
-
-Edit the gateway configuration to match what you used on your client. Specifically, edit the configuration file at:
-
-```
-~/.nym/service-providers/network-requester/mynetworkrequester/config/config.toml
-```
-
-Ensure that the fields `gateway_id`, `gateway_owner`, `gateway_listener` in the new config match those in the old client config at:
-
-```
-~/.nym/clients/myoldclient/config/config.toml
-```
-
-## Automating your network requester with systemd
-Stop the running process with `CTRL-C`, and create a service file for the requester as we did with our client instance previously at `/etc/systemd/system/nym-network-requester.service`:
-
-```ini
-[Unit]
-Description=Nym Network Requester 
-StartLimitInterval=350
-StartLimitBurst=10
-
-[Service]
-User=nym # replace this with whatever user you wish
-LimitNOFILE=65536
-# remember to add the `--enable-statistics` flag if running as part of a service grant and check the path to your nym-network-requester binary
-ExecStart=/home/nym/nym-network-requester run --id <your_id>
-KillSignal=SIGINT
-Restart=on-failure
-RestartSec=30
-
-[Install]
-WantedBy=multi-user.target
-```
-
-Now enable and start your requester:
-
-```
-systemctl enable nym-network-requester.service
-systemctl start nym-network-requester.service
-
-# you can always check your requester has succesfully started with:
-systemctl status nym-network-requester.service
-```
-
-## VPS Setup
-### Configure your firewall
-Although your requester is now ready to receive traffic, your server may not be - the following commands will allow you to set up a properly configured firewall using `ufw`:
-
-```
-# check if you have ufw installed
-ufw version
-# if it is not installed, install with
-sudo apt install ufw -y
-# enable ufw
-sudo ufw enable
-# check the status of the firewall
-sudo ufw status
-```
-
-Finally open your requester's ssh port to incoming administration connections:
-
-```
-sudo ufw allow 22/tcp
-# check the status of the firewall
-sudo ufw status
-```
->>>>>>> release/v1.1.27:documentation/docs/src/nodes/network-requester-setup.md
```

Comment:
Looks like you've left in some merge info

Reply:
the problem was that this part was suppose to be deleted long time ago but due to overlooked merge problem it wasn't. it's all on maintenance page now.